packages/zapp/console/src/components/Executions/TaskExecutionsList/MapTaskExecutionListItem.tsx

```diff
@@ -94,7 +77,21 @@ export const MapTaskExecutionsListItem: React.FC<MapTaskExecutionsListItemProps>
         </section>
       ) : null}
       {/* child/array logs separated by subtasks phase */}
-      {logsSections}
+      {RENDER_ORDER.map((phase, id) => {
+        const logs = logsByPhase.get(phase);
+        if (!logs) {
+          return null;
+        }
+        const key = `${id}-${phase}-${selectedPhase}`;
```

Comment:
nit: I think we don't need `selectedPhase` in this key, as it will re-render items more often then needed? HOwever, I haven't tested my self so not 100% sure.

Reply:
hm, I swear react was complaining on just `id-phase` at some point, so I added extra to it, tested now and it's fine, so removing `selectedPhase`